src/main/java/sc/fiji/bdvpg/bdv/navigate/ViewerTransformSyncStarter.java

```diff
@@ -0,0 +1,90 @@
+package sc.fiji.bdvpg.bdv.navigate;
+
+import bdv.util.BdvHandle;
+import net.imglib2.realtransform.AffineTransform3D;
+import net.imglib2.ui.TransformListener;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Action which synchronizes the display location of n BdvHandle
+ *
+ * Works in combination with the action ViewerTransformSyncStopper
+ *
+ * See also ViewTransformSynchronizationDemo
+ *
+ * author Nicolas Chiaruttini, BIOP EPFL, nicolas.chiaruttini@epfl.ch
+ *
+ */
+
+public class ViewerTransformSyncStarter implements Runnable {
+
+    BdvHandle[] bdvhs;
+
+    BdvHandle originatingBdvHandle = null;
+
+    Map<BdvHandle, TransformListener<AffineTransform3D>> bdvToTransformListener = new HashMap<>();
+
+    public ViewerTransformSyncStarter(BdvHandle[] bdvhs) {
+       this.bdvhs = bdvhs;
+    }
+
+    public void setOriginatingBdvHandle(BdvHandle bdvh) {
+        originatingBdvHandle = bdvh;
+    }
+
+    @Override
+    public void run() {
+
+        // Building circularly linked listeners with stop condition when all transforms are equal
+
+        AffineTransform3D at3Dorigin = null;
+
+        for (int i=0;i<bdvhs.length;i++) {
+            BdvHandle bdvhi = bdvhs[i];
+            if (bdvhi.equals(originatingBdvHandle)) {
+                //System.out.println("Catching origin transform");
+                at3Dorigin = new AffineTransform3D();
+                bdvhi.getViewerPanel().getState().getViewerTransform(at3Dorigin);
+            }
+        }
+
+        for (int i=0;i<bdvhs.length;i++) {
+
+            BdvHandle bdvhip1;
```

Comment:
Also here, could you find a more expressive name than `bdvhip1`?

Reply:
bdvhip1 is `nextBdvHandle`

oki doki! done in https://github.com/bigdataviewer/bigdataviewer-playground/pull/68/commits/72739e1cf201b698fe4dbfb38b982d5515b2e1a4